project/Common.scala

```diff
@@ -51,25 +51,27 @@ object Common extends AutoPlugin {
         else
           scalacOptionsBase
       },
-      javacOptions ++= Seq(
+      Compile / javacOptions ++= Seq(
           "-Xlint:unchecked"
         ),
-      javacOptions ++= (
+      Compile / javacOptions ++= (
           if (isJdk8) Seq.empty
           else Seq("--release", "8")
         ),
       scalacOptions ++= (
           if (isJdk8 || scalaVersion.value == Dependencies.Scala212) Seq.empty
           else Seq("--release", "8")
         ),
+      Compile / doc / javacOptions := Nil,
       Compile / doc / scalacOptions := scalacOptions.value ++ Seq(
           "-doc-title",
           "Akka Management",
           "-doc-version",
-          version.value,
-          "-skip-packages",
-          "akka.pattern" // for some reason Scaladoc creates this
-        ),
+          version.value
+        ) ++
+        // for some reason Scaladoc creates this
+        (if (scalaVersion.value.startsWith("3")) Seq.empty
+         else Seq("-skip-packages", "akka.pattern")),
```

Comment:
This was the problem, flag doesn't seem to exist with Scala 3 (or works differently) and would stop building artifacts for publishing.